cookbooks/aws-parallelcluster-config/resources/volume/volume.rb

```diff
@@ -0,0 +1,130 @@
+provides :volume
+unified_mode true
+
+property :shared_dir, String, required: %i(mount export unmount)
```

Comment:
Nitpick: since the resources seems general for every kind of volume, the name `shared_dir` can be more general